doc/cephadm/services/mgmt-gateway.rst

```diff
@@ -94,9 +100,11 @@ To deploy the mgmt-gateway in a high availability setup, here is an example of t
 
 The number of deployed instances is determined by the number of hosts with the mgmt label.
 The ingress is configured in `keepalive_only` mode, with labels ensuring that any changes to
-the mgmt-gateway daemons are replicated to the corresponding keepalived instances. Additionally,
-the `virtual_ip` parameter must be identical in both specifications.
+the `mgmt-gateway` daemons are replicated to the corresponding keepalived instances.
+
+.. note::
```

Comment:
We might want to even make this a warning instead of a note. I think this is something that would be pretty easy for users to miss

Reply:
yeah, I'll see if we can detect that automatically :+1: (would be a separate PR anyway)

Reply:
Also suggest double backticks around keepalived, and on line 102 above, keepalive_only.